ethexe/runtime/common/src/journal.rs

```diff
@@ -17,15 +17,16 @@ use gear_core::{
     reservation::GasReserver,
 };
 use gear_core_errors::SignalCode;
-use gprimitives::{ActorId, CodeId, MessageId, ReservationId};
+use gprimitives::{ActorId, CodeId, MessageId, ReservationId, H256};
 
-pub struct Handler<'a, S: Storage> {
+// Handles unprocessed journal notes during chunk processing.
+pub struct ChunkJournalHandler<'a, S: Storage> {
```

Comment:
I'd highlight usage of these handlers by naming: NativeJournalHandler and RuntimeJournalHandler

Reply:
Moreover it handles not chunk, but a single program

Reply:
Renamed